app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -986,51 +1031,6 @@ public String getActivityTitle() {
     protected boolean isTopNavEnabled() {
         return false;
     }
-
-    private void formEntry(Uri formUri, FormRecord r) throws SessionUnavailableException{
-        formEntry(formUri, r, null);
-    }
-    
-    private void formEntry(Uri formUri, FormRecord r, String headerTitle) throws SessionUnavailableException{
-        Logger.log(AndroidLogger.TYPE_FORM_ENTRY, "Form Entry Starting|" + r.getFormNamespace());
```

Comment:
Removed this logger line since it seems like a not very useful thing to log.. Is that cool?
What is the protocol behind logging?